prime-router/src/main/resources/db/migration/V20__add_sftp_check_result_table.sql

```diff
@@ -0,0 +1,35 @@
+/*
+ * The Flyway tool applies this migration to create the database.
+ *
+ * Follow this style guide https://about.gitlab.com/handbook/business-ops/data-team/platform/sql-style-guide/
+ * use VARCHAR(63) for names in organization and schema
+ *
+ * Copy a version of this comment into the next migration
+ *
+ */
+
+CREATE TABLE receiver_connection_check_results
+(
+  connection_check_result_id BIGSERIAL PRIMARY KEY,
```

Comment:
To match naming conventions of other tables in the database this should be `receiver_connection_check_results_id`